src/solvers/stack_decision_procedure.h

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************\
+
+Module: Decision procedure with incremental solving with contexts
+        and assumptions
+
+Author: Peter Schrammel
+
+\*******************************************************************/
+
+/// \file
+/// Decision procedure with incremental solving with contexts
```

Comment:
Perhaps give an example of intended usage?

Reply:
I've added two examples